compiler/src/iree/compiler/Dialect/VM/Conversion/StandardToVM/test/conversion_ops.mlir

```diff
@@ -77,7 +77,7 @@ module @my_module {
 module @t006_fptosi_fp32_i8 {
 module @my_module {
   func.func @my_fn(%arg0 : f32) -> (i8) {
-// expected-error@+1 {{failed to legalize}}
+    // CHECK: vm.cast.f32.si32 %[[ARG0]] : f32 -> i32
     %1 = arith.fptosi %arg0 : f32 to i8
     return %1 : i8
```

Comment:
Are there other IRs that truncate i32 to i8? If so, should we check it as well?

Reply:
We checked the integer ones yesterday (https://github.com/iree-org/iree/commit/8a3d270d393a22519fec356fb5ec418afcd3e30c). At least for those individually the result wrt llvm and vmvx was the same.